libs/wingsdk/src/target-sim/redis.inflight.ts

```diff
@@ -25,13 +26,17 @@ export class Redis implements IRedisClient, ISimulatorResourceInstance {
       "."
     )}-${uuidv4()}`;
   }
-
+  
   public async init(): Promise<RedisAttributes> {
     this.docker = new Dockerode();
+
+    // Pull docker image
+    const stream = await this.docker.pull(this.REDIS_IMAGE);
+    await new Promise(res => this.docker?.modem.followProgress(stream, res));
```

Comment:
should these go into the `try`? I guess we also need to fix the error message to only suggest installing docker since we're taking care of the image.